targets.json

```diff
@@ -2479,6 +2479,20 @@
             }
         }
     },
+    "spracing": {
+      "name": "SPRacing",
+      "rx_2400": {
+        "rxn1": {
+          "product_name": "SPRacing RXN1 Gyro 2.4GHz RX/GYRO/IR (D4)",
+          "lua_name": "SPR_RXN1_D4",
+          "layout_file": "SPRacing RXN1 Gyro 2400 RX GYRO IR D4.json",
+          "upload_methods": ["uart", "wifi", "betaflight"],
+          "platform": "esp32",
+          "min_version": "3.3.0",
```

Comment:
I suppose it can be 3.3.0 without the Gyro support and when it comes it will just work!

Reply:
I think the current release 3.3 is probably the best bet. Just because it's a PWM device and there have been quite a few PWM fixes added in the later versions.